publish.js

```diff
@@ -1,93 +1,101 @@
 var fs = require('fs');
 var path = require('path');
 
+var R = require('ramda');
+
+var {defaultTo, map, pipe, prop} = R
+
 var handlebars = require('handlebars');
 var hljs = require('highlight.js');
 var helper = require('jsdoc/util/templateHelper');
 var marked = require('marked');
 
 var version = require('./package.json').devDependencies.ramda
 
-var headOr = function(x, xs) {
-  return xs.length === 0 ? x : xs[0];
-};
-
-var chain = function(f, xs) {
-  var result = [];
-  for (var idx = 0; idx < xs.length; idx += 1) {
-    result = result.concat(f(xs[idx]));
-  }
-  return result;
-};
-
-var valuesForTitle = function(title, xs) {
-  var result = [];
-  for (var idx = 0; idx < xs.length; idx += 1) {
-    if (xs[idx].title === title) {
-      result.push(xs[idx].value);
-    }
-  }
-  return result;
-};
 
-var prettifySig = function(s) {
-  return s.replace(/[.][.][.]/g, '\u2026').replace(/->/g, '\u2192');
-};
+var prettifyCode = R.pipe(
+  R.join('\n'),
+  R.replace(/^[ ]{5}/gm, ''),
+  R.flip(R.invoker(2, 'highlight')('javascript'))(hljs),
```

Comment:
If we're to use a fat arrow here, why not throughout the file? While we're at it we could replace occurrences of `var` with `const`. :)